### PR TITLE
Update health assessment interval

### PR DESCRIPTION
**Pull request by BrianMMcClain**
----------

### Merge Checklist
_If items do not apply to your changes, add (N/A) and mark them as complete._

#### Pull Request
- [x] One or more labels describe the type of change (e.g. clarification) and associated product (e.g. HCP Terraform ).
- [x] Description links to related pull requests or issues, if any.

#### Content
- [x] Redirects have been added to `website/redirects.js` for moved, renamed, or deleted pages.
- [x] API documentation and the API Changelog have been updated. 
- [x] Links to related content where appropriate (e.g., API endpoints, permissions, etc.).
- [x] Pages with related content are updated and link to this content when appropriate.
- [x] Sidebar navigation files have been updated for added, deleted, reordered, or renamed pages.
- [x] New pages have metadata (page name and description) at the top.
- [x] New images are 2048 px wide. They have HashiCorp standard annotation color (#F92672) and format (rectangle with rounded corners), blurred sensitive details (e.g. credentials, usernames, user icons), and descriptive alt text in the markdown for accessibility.
- [x] New code blocks have the correct syntax and line breaks to eliminate horizontal scroll bars.
- [x] UI elements (button names, page names, etc.) are bolded.
- [x] The Vercel website preview successfully deployed.

#### Reviews
- [x] I or someone else reviewed the content for technical accuracy.
- [x] I or someone else reviewed the content for typos, punctuation, spelling, and grammar.

### CODE REVIEW

File: website/docs/cloud-docs/workspaces/health.mdx [hunk 1]
```diff
@@ -11,7 +11,7 @@ HCP Terraform can perform automatic health assessments in a workspace to assess
 - [Drift detection](#drift-detection) determines whether your real-world infrastructure matches your Terraform configuration.
 - [Continuous validation](#continuous-validation) determines whether custom conditions in the workspace’s configuration continue to pass after Terraform provisions the infrastructure.
 
-When enabled, HCP Terraform automatically runs a health assessment for your workspace about every 4 hours. Refer to [Health Assessment Scheduling](#health-assessment-scheduling) for details.
+When enabled, HCP Terraform periodically runs an automated health assessment for your workspace. Refer to [Health Assessment Scheduling](#health-assessment-scheduling) for details.
```
Comment (robin-norwood) [inline website/docs/cloud-docs/workspaces/health.mdx:14]:
```suggestion
When you enable health assessments, HCP Terraform periodically runs health assessments for your workspace. Refer to [Health Assessment Scheduling](#health-assessment-scheduling) for details.
```

Remove passive voice, mention the feature by name. Although now it says "health assessment" twice, I think it scans ok?

File: website/docs/cloud-docs/workspaces/health.mdx [hunk 2]
```diff
@@ -20,14 +20,15 @@ When enabled, HCP Terraform automatically runs a health assessment for your work
 ## Permissions
 
 Working with health assessments requires the following permissions:
+
 - To view health status for a workspace, you need read access to that workspace.
 - To change organization health settings, you must be an [organization owner](/terraform/cloud-docs/users-teams-organizations/permissions#organization-owners).
 - To change a workspace’s health settings, you must be an [administrator for that workspace](/terraform/cloud-docs/users-teams-organizations/permissions#workspace-admins).
 <!-- BEGIN: TFC:only name:health-assessments -->
-- To trigger [on demand assessments](/terraform/cloud-docs/workspaces/health#on-demand-assessments) for a workspace, you must be an [administrator for that workspace](/terraform/cloud-docs/users-teams-organizations/permissions#workspace-admins).
+- To trigger [on-demand assessments](/terraform/cloud-docs/workspaces/health#on-demand-assessments) for a workspace, you must be an [administrator for that workspace](/terraform/cloud-docs/users-teams-organizations/permissions#workspace-admins).
```
Comment (robin-norwood) [inline website/docs/cloud-docs/workspaces/health.mdx:28]:
```suggestion
- To trigger [on-demand assessments](/terraform/cloud-docs/workspaces/health#on-demand-assessments) health assessments for a workspace, you must be an [administrator for that workspace](/terraform/cloud-docs/users-teams-organizations/permissions#workspace-admins).
```

File: website/docs/cloud-docs/workspaces/health.mdx [hunk 3]
```diff
@@ -49,32 +50,35 @@ To enable health assessments within a workspace:
 1. Select **Enable** under **Health Assessments**.
 1. Click **Save settings**.
 
-## Health Assessment Scheduling
+## Health assessment scheduling
 
 The timing of the first health assessment in the workspace depends on whether you enable health assessments during active Terraform runs:
```
Comment (robin-norwood) [inline website/docs/cloud-docs/workspaces/health.mdx:55]:
```suggestion
When you enable health assessments for a workspace, Terraform Cloud will run the first health assessment based on whether there are active Terraform runs for the workspace:
```

Comment (robin-norwood) [reply website/docs/cloud-docs/workspaces/health.mdx:55]: Remove passive voice.

File: website/docs/cloud-docs/workspaces/health.mdx [hunk 4]
```diff
@@ -49,32 +50,35 @@ To enable health assessments within a workspace:
...
+
 - **No active runs:** The first health assessment starts a few minutes after you enable the feature.
 - **Active speculative plan:** The first health assessment starts soon after that plan's completion.
```
Comment (robin-norwood) [inline website/docs/cloud-docs/workspaces/health.mdx:58]:
```suggestion
- **No active runs:** A few minutes after you enable the feature.
- **Active speculative plan:** A few minutes after that plan is complete.
```

File: website/docs/cloud-docs/workspaces/health.mdx [hunk 5]
```diff
@@ -49,32 +50,35 @@ To enable health assessments within a workspace:
...
-- **Other active runs:** The first health assessment starts in about 4 hours.
+- **Other active runs:** The first health assessment starts during the next assessment period.
```
Comment (robin-norwood) [inline website/docs/cloud-docs/workspaces/health.mdx:59]:
```suggestion
- **Other active runs:** During the next assessment period.
```

File: website/docs/cloud-docs/workspaces/health.mdx [hunk 6]
```diff
@@ -49,32 +50,35 @@ To enable health assessments within a workspace:
...
+### On-demand assessments
+
+-> **Note:** On-demand assessments are only available in the HCP Terraform web UI.
 
 If you are an administrator for a workspace and it satisfies all [assessment requirements](/terraform/cloud-docs/workspaces/health#workspace-requirements), you can trigger a new assessment by clicking **Start health assessment** on the workspace's **Health** page.
 
 When you click the **Start health assessment** button, the workspace displays a message in the bottom lefthand corner of the page to indicate if it successfully triggered a new assessment. The time it takes to complete an assessment can vary based on network latency and the number of resources managed by the workspace. 
 
-While an assessment is in progress, you cannot trigger another assessment. And on demand assessment resets the scheduling for automated assessments, so the next automated assessment will run in 4 hours. 
+While an assessment is in progress, you cannot trigger another assessment. An on-demand assessment resets the scheduling for automated assessments, so the next automated assessment will during the next assessment period.
```
Comment (robin-norwood) [inline website/docs/cloud-docs/workspaces/health.mdx:80]:
```suggestion
While an assessment is in progress, you cannot trigger another assessment. An on-demand assessment resets the scheduling for automated assessments, so Terraform Cloud will run the next assessment during the next assessment period. 
```

File: website/docs/cloud-docs/workspaces/health.mdx [hunk 7]
```diff
@@ -49,32 +50,35 @@ To enable health assessments within a workspace:
...
 
 Terraform Enterprise administrators can modify their installation's [assessment frequency  and number of maximum concurrent assessments](/terraform/enterprise/admin/application/general#health-assessments) from the admin settings console.
 
 <!-- BEGIN: TFC:only name:health-assessments -->
-### On Demand Assessments
 
--> **Note:** On demand assessments are only available in the HCP Terraform web UI.
+### On-demand assessments
+
+-> **Note:** On-demand assessments are only available in the HCP Terraform web UI.
```
Comment (rkoron007) [inline website/docs/cloud-docs/workspaces/health.mdx:74]: I like spelling out acronyms when we can:
```suggestion
-> **Note:** On-demand assessments are only available in the HCP Terraform user interface.
```

File: website/docs/cloud-docs/workspaces/health.mdx [hunk 8]
```diff
@@ -49,32 +50,35 @@ To enable health assessments within a workspace:
...
 
 If you are an administrator for a workspace and it satisfies all [assessment requirements](/terraform/cloud-docs/workspaces/health#workspace-requirements), you can trigger a new assessment by clicking **Start health assessment** on the workspace's **Health** page.
 
 When you click the **Start health assessment** button, the workspace displays a message in the bottom lefthand corner of the page to indicate if it successfully triggered a new assessment. The time it takes to complete an assessment can vary based on network latency and the number of resources managed by the workspace.
```
Comment (rkoron007) [inline website/docs/cloud-docs/workspaces/health.mdx:78]:
```suggestion
After clicking **Start health assessment**, the workspace displays a message in the bottom lefthand corner of the page to indicate if it successfully triggered a new assessment. The time it takes to complete an assessment can vary based on network latency and the number of resources managed by the workspace. 
```

File: website/docs/cloud-docs/workspaces/health.mdx [hunk 9]
```diff
@@ -49,32 +50,35 @@ To enable health assessments within a workspace:
...
+### On-demand assessments
+
+-> **Note:** On-demand assessments are only available in the HCP Terraform web UI.
 
 If you are an administrator for a workspace and it satisfies all [assessment requirements](/terraform/cloud-docs/workspaces/health#workspace-requirements), you can trigger a new assessment by clicking **Start health assessment** on the workspace's **Health** page.
 
 When you click the **Start health assessment** button, the workspace displays a message in the bottom lefthand corner of the page to indicate if it successfully triggered a new assessment. The time it takes to complete an assessment can vary based on network latency and the number of resources managed by the workspace. 
 
-While an assessment is in progress, you cannot trigger another assessment. And on demand assessment resets the scheduling for automated assessments, so the next automated assessment will run in 4 hours. 
+While an assessment is in progress, you cannot trigger another assessment. An on-demand assessment resets the scheduling for automated assessments, so HCP Terraform will run the next assessment during the next assessment period.
```
Comment (rkoron007) [inline website/docs/cloud-docs/workspaces/health.mdx:80]: Little tweaks for present tense
```suggestion
You cannot trigger another assessment while one is in progress. An on-demand assessment resets the scheduling for automated assessments, so HCP Terraform waits to run the next assessment until the next scheduled period.
```

**Review by rkoron007 — APPROVED** [review]
Looks great! Thank you for making this! ✨

**Pull request merged**